### PR TITLE
Remove extraneous assignment in kill-emulator

### PR DESCRIPTION
**Pull request by mwleeds**

### CODE REVIEW

**Comment by mdimjasevic** [comment]
Hi @mwleeds,

Thanks for showing interest in maline!

Would you be so kind to describe the pull request?

**Comment by mwleeds** [comment]
Sure. It appears that everywhere kill-emulator is called the console port is the only argument. The extra assignment to name was probably just accidentally copied over from avd-reload.

By the way, thanks for maline! I just submitted a paper to a conference that made use of it.

**Comment by mdimjasevic** [comment]
Wow, that's great to hear! Would you say in a sentence or two how you made use of it?

**Comment by mwleeds** [comment]
I used it to get lists of system calls used by a number of apps, which I then used to train a machine learning model (just a simple TensorFlow softmax one), but it only achieved a classification accuracy of around 60%. Possibly because the sample size was 200 apps. So I think pretty similar to what you did, but with a different ML model and fewer apps. I also looked at a model that was trained on permission requests rather than system calls in the same paper, which had a better accuracy (> 80%).

**Comment by mdimjasevic** [comment]
I'll take a look at this. It's been a while since we ran maline.
